test/fixtures/v5/off_ramp/uturn_exit.json

```diff
@@ -0,0 +1,22 @@
+{
+    "step": {
+        "maneuver": {
+            "type": "off ramp",
+            "modifier": "uturn"
+        },
+        "name": "Way Name",
+        "exits": "4A,4B"
+    },
+    "instructions": {
+        "de": "Rampe nehmen auf Way Name",
```

Comment:
German nitpick - are these really called "Rampe" and not rather "Ausfahrt"? cc @freenerd